sdk/ml/azure-ai-ml/azure/ai/ml/entities/_schedule/schedule.py

```diff
@@ -218,7 +220,7 @@ def _from_rest_object(cls, obj: RestSchedule) -> "JobSchedule":
                 target=ErrorTarget.JOB,
                 error_category=ErrorCategory.SYSTEM_ERROR,
             )
-        if camel_to_snake(action.job_definition.job_type) != JobType.PIPELINE:
+        if camel_to_snake(action.job_definition.job_type) not in [JobType.PIPELINE, JobType.COMMAND]:
```

Comment:
we should add private preview feature check some where

Reply:
add into _to_rest_object